CesiumRasterOverlays/src/RasterOverlayUtilities.cpp

```diff
@@ -339,4 +517,1138 @@ RasterOverlayUtilities::createRasterOverlayTextureCoordinates(
+            edgeIndices,
+            0.5,
+            0.5,
+            keepAboveU,
+            keepAboveV,
+            hasInvertedVCoordinate,
+            uvView,
+            clipVertexToIndices,
+            clippedA,
+            clippedB);
+      }
+    }
+  }
+
+  // create mesh with skirt
+  std::optional<SkirtMeshMetadata> skirtMeshMetadata;
+  if (hasSkirt) {
+    skirtMeshMetadata = std::make_optional<SkirtMeshMetadata>();
+    skirtMeshMetadata->noSkirtIndicesBegin = 0;
+    skirtMeshMetadata->noSkirtIndicesCount =
+        static_cast<uint32_t>(indices.size());
+    skirtMeshMetadata->noSkirtVerticesBegin = 0;
+    skirtMeshMetadata->noSkirtVerticesCount =
+        uint32_t(newVertexFloats.size() / size_t(vertexSizeFloats));
+    skirtMeshMetadata->meshCenter = parentSkirtMeshMetadata->meshCenter;
+    addSkirts(
+        newVertexFloats,
+        indices,
+        attributes,
+        childID,
+        *skirtMeshMetadata,
+        *parentSkirtMeshMetadata,
+        edgeIndices,
+        vertexSizeFloats,
+        positionAttributeIndex,
+        hasInvertedVCoordinate);
+  }
+
+  if (newVertexFloats.empty() || indices.empty()) {
+    return false;
+  }
+
+  // Update the accessor vertex counts and min/max values
+  const int64_t numberOfVertices =
+      int64_t(newVertexFloats.size()) / vertexSizeFloats;
+  for (const FloatVertexAttribute& attribute : attributes) {
+    Accessor& accessor =
+        model.accessors[static_cast<size_t>(attribute.accessorIndex)];
+    accessor.count = numberOfVertices;
+    accessor.min = std::move(attribute.minimums);
+    accessor.max = std::move(attribute.maximums);
+  }
+
+  // Add an accessor for the indices
+  const size_t indexAccessorIndex = model.accessors.size();
+  model.accessors.emplace_back();
+  Accessor& newIndicesAccessor = model.accessors.back();
+  newIndicesAccessor.bufferView = static_cast<int>(indexBufferViewIndex);
+  newIndicesAccessor.byteOffset = 0;
+  newIndicesAccessor.count = int64_t(indices.size());
+  newIndicesAccessor.componentType = Accessor::ComponentType::UNSIGNED_INT;
+  newIndicesAccessor.type = Accessor::Type::SCALAR;
+
+  // Populate the buffers
+  Buffer& vertexBuffer = model.buffers[vertexBufferIndex];
+  vertexBuffer.cesium.data.resize(newVertexFloats.size() * sizeof(float));
+  float* pAsFloats = reinterpret_cast<float*>(vertexBuffer.cesium.data.data());
+  std::copy(newVertexFloats.begin(), newVertexFloats.end(), pAsFloats);
+  vertexBuffer.byteLength = vertexBufferView.byteLength =
+      int64_t(vertexBuffer.cesium.data.size());
+  vertexBufferView.byteStride = vertexSizeFloats * int64_t(sizeof(float));
+
+  Buffer& indexBuffer = model.buffers[indexBufferIndex];
+  indexBuffer.cesium.data.resize(indices.size() * sizeof(uint32_t));
+  uint32_t* pAsUint32s =
+      reinterpret_cast<uint32_t*>(indexBuffer.cesium.data.data());
+  std::copy(indices.begin(), indices.end(), pAsUint32s);
+  indexBuffer.byteLength = indexBufferView.byteLength =
```

Comment:
This looks like the fix mentioned in PR description.
(marking for easy lookup later)